src/Command/Humbug.php

```diff
@@ -10,32 +10,37 @@
 
 namespace Humbug\Command;
 
-use Humbug\Adapter\AdapterAbstract;
-use Humbug\Collector;
 use Humbug\Config;
-use Humbug\Config\JsonParser;
 use Humbug\Container;
-use Humbug\Mutant;
-use Humbug\ProcessRunner;
-use Humbug\Report\Text as TextReport;
-use Humbug\Utility\Performance;
-use Humbug\Utility\ParallelGroup;
-use Humbug\Renderer\Text;
+use Humbug\Adapter\Phpunit;
+use Humbug\Config\JsonParser;
 use Humbug\Exception\InvalidArgumentException;
```

Comment:
I'm curious as to why it shouldn't be committed ? Isn't it the equivalent of phpunit.xml or any other test config ?